Nosach/web-quiz-project/src/main/java/dao/QuizDAOImpl.java

```diff
@@ -4,16 +4,22 @@
 import entity.Quiz;
 import entity.Subject;
 import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 import service.SubjectService;
 
 import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
+@Component
 public class QuizDAOImpl implements QuizDAO {
 
     private static final Logger logger = Logger.getLogger(QuizDAOImpl.class);
 
+    @Autowired
+    SubjectService ss;
+
```

Comment:
better to name subjectService, for more readability.

Reply:
Ok, will do